src/components/timecard/timecard-entries-list/TimecardEntriesList.spec.jsx

```diff
@@ -22,7 +22,7 @@ describe('TimecardEntriesList', () => {
     expect(screen.queryByText('Add a new time period')).toBeFalsy();
     expect(screen.getByText('Start time')).toBeTruthy();
     expect(screen.getByText('Finish time')).toBeTruthy();
-    expect(screen.getByText('Shift')).toBeTruthy();
```

Comment:
Could you not still keep this since the test is specifically testing for when the period type is shift?

Reply:
Ah ok, yes that makes sense, was just curious. Will resolve.